Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/webpack.prod.js

```diff
@@ -84,18 +85,17 @@ module.exports = {
         }),
         new webpack.optimize.CommonsChunkPlugin(
             {

```

Comment:
Yes, but you have to use _angular-router-loader_ plugin with webapck, or the Closure Compiler. Rollup doesn't work yet.